cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerModificationService.java

```diff
@@ -136,6 +136,12 @@ public void initApiClient() throws ClusterClientInitException {
                 List<ApiHost> hosts = clouderaManagerApiFactory.getHostsResourceApi(apiClient).readHosts(null, null, SUMMARY).getItems();
                 ApiHostRefList body = createUpscaledHostRefList(upscaleHostNames, hosts);
                 clustersResourceApi.addHosts(clusterName, body);
+                // TODO Eventually (CM): activateParcel takes at least 20 seconds in CM.
```

Comment:
We had all sorts of issues when we didn't invoke the activate parcel. What we did to make it faster is to pre-calculate the parcel hash on the image and replace the parcel itself with a .skiphash file so CM does not try to read its content.

Reply:
OK. Think this can be worked on further once the CM team has some time. Did some more multi-node tests - and at least this is independent of the number of nodes.